PostHog/PostHogSDK.swift

```diff
@@ -831,6 +847,146 @@ let maxRetryDelay = 30.0
     }
 
     // FEATURE FLAGS
+
+    /// Sets person properties that will be included in feature flag evaluation requests.
+    ///
+    /// This method allows you to override server-side person properties for immediate feature flag evaluation,
+    /// solving the race condition where person properties from `identify()` calls may not have been processed
+    /// by the server yet.
+    ///
+    /// Properties are merged additively with existing properties. Feature flags are automatically reloaded
+    /// after setting properties.
+    ///
+    /// ## Example Usage
+    /// ```swift
+    /// // Set properties and automatically reload flags
+    /// PostHogSDK.shared.setPersonPropertiesForFlags([
+    ///     "$app_version": "2.93.0",
+    ///     "plan": "premium"
+    /// ])
+    ///
+    /// // Now feature flags will be evaluated with these properties
+    /// let flagValue = PostHogSDK.shared.isFeatureEnabled("new_feature")
+    /// ```
+    ///
+    /// - Parameter properties: Dictionary of person properties to include in flag evaluation
+    /// - SeeAlso: `setPersonPropertiesForFlags(_:reloadFeatureFlags:)` to control flag reloading behavior
+    @objc public func setPersonPropertiesForFlags(_ properties: [String: Any]) {
+        setPersonPropertiesForFlags(properties, reloadFeatureFlags: true)
+    }
+
+    /// Sets person properties that will be included in feature flag evaluation requests.
+    ///
+    /// This method allows you to override server-side person properties for immediate feature flag evaluation,
+    /// solving the race condition where person properties from `identify()` calls may not have been processed
+    /// by the server yet.
+    ///
+    /// Properties are merged additively with existing properties.
+    ///
+    /// ## Example Usage
+    /// ```swift
+    /// // Set properties without automatically reloading flags
+    /// PostHogSDK.shared.setPersonPropertiesForFlags([
+    ///     "$app_version": "2.93.0",
+    ///     "plan": "premium"
+    /// ], reloadFeatureFlags: false)
+    ///
+    /// // Manually reload flags later
+    /// PostHogSDK.shared.reloadFeatureFlags()
+    /// ```
+    ///
+    /// - Parameters:
+    ///   - properties: Dictionary of person properties to include in flag evaluation
+    ///   - reloadFeatureFlags: Whether to automatically reload feature flags after setting properties
+    @objc(setPersonPropertiesForFlags:reloadFeatureFlags:)
+    public func setPersonPropertiesForFlags(_ properties: [String: Any], reloadFeatureFlags: Bool) {
```

Comment:
we can make `reloadFeatureFlags = true` as default so you dont have to pass the value, less code.
We should also add an overload for this method for better objC UX only with the `properties` param.